tests/functional/timer/test_job_resume.py

```diff
@@ -133,50 +225,5 @@ def _register_responses(mock_user_data):
     )
 
 
-def test_resume_job_active(run_line):
-    TIMERS_GET_RESPONSES.activate("default")
-    TIMERS_RESUME_RESPONSES.activate("default")
-    job_id = TIMERS_GET_RESPONSES.metadata["job_id"]
-    run_line(
-        ["globus", "timer", "resume", job_id],
-        search_stdout=f"Successfully resumed job {job_id}.",
-    )
-
-
-def test_resume_job_inactive_user(run_line):
-    TIMERS_GET_RESPONSES.activate("inactive_user")
-    TIMERS_RESUME_RESPONSES.activate("default")
-    job_id = TIMERS_GET_RESPONSES.metadata["job_id"]
-    run_line(
-        ["globus", "timer", "resume", job_id],
-        search_stdout=f"Successfully resumed job {job_id}.",
-    )
-
-
-def test_resume_job_inactive_gare_consent_missing(run_line):
-    meta = load_response_set("cli.timer_resume.inactive_gare.consents_missing").metadata
-    job_id = meta["job_id"]
-    required_scope = meta["required_scope"]
-    result = run_line(
-        ["globus", "timer", "resume", job_id],
-        assert_exit_code=4,
-    )
-    assert f"globus session consent '{required_scope}'" in result.output
-
-
-def test_resume_job_inactive_gare_consent_present(run_line):
-    meta = load_response_set("cli.timer_resume.inactive_gare.consents_present").metadata
-    job_id = meta["job_id"]
-    run_line(
-        ["globus", "timer", "resume", job_id],
-        search_stdout=f"Successfully resumed job {job_id}.",
-    )
-
-
-def test_resume_job_inactive_gare_consent_missing_but_skip_check(run_line):
-    meta = load_response_set("cli.timer_resume.inactive_gare.consents_missing").metadata
-    job_id = meta["job_id"]
-    run_line(
-        ["globus", "timer", "resume", "--skip-inactive-reason-check", job_id],
-        search_stdout=f"Successfully resumed job {job_id}.",
-    )
+def _urlscope(m: str, s: str) -> str:
+    return f"https://auth.globus.org/scopes/{m}/{s}"
```

Comment:
This is called twice from two adjacent lines in the same function. Would it be beneficial to replace the function calls with a string literal and an f-string?

Reply:
I sort of think this would be a nice test utility to just have on hand in some central place.
But it's out of scope -- the only reason this line appears as changed is that I reordered file contents.